src/signals/settings/users/containers/Overview/index.js

```diff
@@ -88,22 +97,26 @@ const UsersOverviewContainer = () => {
     [createOnChangeFilter]
   );
 
+  const selectUserActiveOnChange = useCallback(event => {
+    event.preventDefault();
+    dispatch(setUserFilters({ is_active: event.target.value }));
+  }, [dispatch]);
+
+  const selectRoleOnChange = useCallback(event => {
+    event.preventDefault();
+    dispatch(setUserFilters({ role: event.target.value }));
+  }, [dispatch]);
+
   const onItemClick = useCallback(
-    e => {
+    event => {
       if (userCan('change_user') === false) {
-        e.preventDefault();
+        event.preventDefault();
         return;
       }
 
-      const {
-        currentTarget: {
-          dataset: { itemId },
-        },
-      } = e;
+      const { currentTarget: { dataset: { itemId } } } = event;
 
-      if (itemId) {
-        history.push(`${USER_URL}/${itemId}`);
-      }
+      if (itemId) { history.push(`${USER_URL}/${itemId}`); }
```

Comment:
Does the linter turn this into a one-liner?

Reply:
My linter in my mind does, such a short line which took 3 lines, I would even prefer not to use the curly braces and semicolon.

Will change it back..